frontend/src/components/shared/ProgressBar/ProgressBar.tsx

```diff
@@ -68,7 +68,7 @@ function ProgressBar({
     },
     Bar: {
       style: ({ $theme }: { $theme: any }) => ({
-        width,
+        width: width ? width.toString() : "100%",
```

Comment:
Was the value from `width` changed to string instead of int in baseUI? What is the reason it needs the `100%` now as fallback, would it also be an option to not set `width` at all when it is empty?

Reply:
Now `baseui` needs width as a `string`, but it can also accept `undefined` value, good call we should pass `undefined` instead of `"100%` 👍 I've changed that